src/main/java/org/spongepowered/api/command/CommandResult.java

```diff
@@ -260,18 +206,27 @@ public Builder affectedItems(@Nullable Integer affectedItems) {
          * @param queryResult The query result of the command
          * @return This builder, for chaining
          */
-        public Builder queryResult(@Nullable Integer queryResult) {
-            this.queryResult = queryResult;
-            return this;
-        }
+        Builder queryResult(@Nullable Integer queryResult);
+
+        /**
+         * Sets or unsets the error message to return to the user without
+         * throwing an exception.
+         *
+         * <p>If this is set, then the command parser will send this message to
+         * the command invoker.</p>
+         *
+         * @param errorMessage The message to send to the user.
+         * @return This builder, for chaining
+         */
+        Builder error(@Nullable Text errorMessage);
```

Comment:
Does the caller have to apply the error formatting or is the command manager supposed to do that.